toolkits/math/evals/eval_math_tools.py

```diff
@@ -25,7 +25,7 @@
 def math_eval_suite():
     suite = EvalSuite(
         name="Math Tools Evaluation",
-        system_message="You are an AI assistant with access to math tools. Use them to help the user with their math-related tasks.",
+        system_message="You're an AI assistant with access to math tools. Use them to help the user with their math-related tasks.",
```

Comment:
adding this minor change to ensure the 'Check Version' step that I added is working as intended.